boards.txt

```diff
@@ -51572,4 +51572,216 @@ fed4.menu.ZigbeeMode.zczr=Zigbee ZCZR (coordinator/router)
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.160=160MHz (WiFi)
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.160.build.f_cpu=160000000L
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.80=80MHz (WiFi)
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.80.build.f_cpu=80000000L
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.40=40MHz
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.40.build.f_cpu=40000000L
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.20=20MHz
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.20.build.f_cpu=20000000L
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.10=10MHz
+ fobe_quill_esp32s3_mesh.menu.CPUFreq.10.build.f_cpu=10000000L
+
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.921600=921600
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.921600.upload.speed=921600
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.115200=115200
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.115200.upload.speed=115200
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.256000.windows=256000
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.256000.upload.speed=256000
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.230400.windows.upload.speed=256000
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.230400=230400
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.230400.upload.speed=230400
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.460800.linux=460800
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.460800.macosx=460800
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.460800.upload.speed=460800
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.512000.windows=512000
+ fobe_quill_esp32s3_mesh.menu.UploadSpeed.512000.upload.speed=512000
+
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.none=None
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.none.build.code_debug=0
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.error=Error
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.error.build.code_debug=1
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.warn=Warn
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.warn.build.code_debug=2
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.info=Info
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.info.build.code_debug=3
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.debug=Debug
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.debug.build.code_debug=4
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.verbose=Verbose
+ fobe_quill_esp32s3_mesh.menu.DebugLevel.verbose.build.code_debug=5
+
+ fobe_quill_esp32s3_mesh.menu.EraseFlash.none=Disabled
+ fobe_quill_esp32s3_mesh.menu.EraseFlash.none.upload.erase_cmd=
+ fobe_quill_esp32s3_mesh.menu.EraseFlash.all=Enabled
+ fobe_quill_esp32s3_mesh.menu.EraseFlash.all.upload.erase_cmd=-e
+
+ fobe_quill_esp32s3_mesh.menu.ZigbeeMode.default=Disabled
+ fobe_quill_esp32s3_mesh.menu.ZigbeeMode.default.build.zigbee_mode=
+ fobe_quill_esp32s3_mesh.menu.ZigbeeMode.default.build.zigbee_libs=
+ fobe_quill_esp32s3_mesh.menu.ZigbeeMode.zczr=Zigbee ZCZR (coordinator/router)
+ fobe_quill_esp32s3_mesh.menu.ZigbeeMode.zczr.build.zigbee_mode=-DZIGBEE_MODE_ZCZR
+ fobe_quill_esp32s3_mesh.menu.ZigbeeMode.zczr.build.zigbee_libs=-lesp_zb_api.zczr -lzboss_stack.zczr -lzboss_port.remote
```

Comment:
You have there the Zigbee mode, but you missed the Zigbee ZCZR partition scheme. Add the partition or remove the Zigbee mode.